### PR TITLE
Initial version of documentation and manpage

### PR DESCRIPTION
**Pull request by tenko**
This adds a start of documentation and manpage for the project.

I started on converting the Oberon-07 language report to the restructuredtext format.
This will take some time to finish.

Also the manpage looks ok for me, but there could be errors visible on other platforms.

I mark this as [DRAFT] as I wanted some feedback if this is in line with what you where planning here.
Later when this is more complete this can be added to the CI and automatic upload to the gh-pages branch.

### CODE REVIEW

**Comment by zaskar9** [comment]
> This adds a start of documentation and manpage for the project.

Great! Very much appreciated! 🙏🏻

> I started on converting the Oberon-07 language report to the restructuredtext format. This will take some time to finish.

Is that required? Can't we just link Wirth's original document? I think what would be really useful down the line is to combine all these language reports (Oberon, Oberon-2, Oberon-07, Oberon+, etc.) into one consistent document that shows where they differ and where `oberon-lang` makes extensions. Maybe converting Wirth's report can be a good basis for that, even though it is a bit wordy.

> I mark this as [DRAFT] as I wanted some feedback if this is in line with what you where planning here. Later when this is more complete this can be added to the CI and automatic upload to the gh-pages branch.

Awesome!

**Comment by tenko** [comment]
Very good.
I agree that involving the language report at this stage might be premature.

Updated:
* Removed language report
* Updated manpage to render correctly and added comment how to test render the page
* Added an entry to the FAQ and added a local TOC tree for the page

I think that this is enough for an initial minimal version of the documentation.
